lib/styles/editor/_character-count.scss

```diff
@@ -0,0 +1,6 @@
+.neeto-editor-character-count {
```

Comment:
Can we have these styles inside the editor.scss itself? This is a tiny element which doesn't seem to need a stylesheet of it's own